src/art/art.cpp.old

```diff
@@ -0,0 +1,633 @@
+/*                     A R T . C P P
```

Comment:
Is there a reason for storing a complete copy of art.cpp.old into the repo?  Git has the history if we need to roll back.